configure.ac

```diff
@@ -1693,6 +1698,18 @@ AS_IF([test x"$enable_naked_pointers" = "xno" ],
    AC_DEFINE([NO_NAKED_POINTERS])],
   [naked_pointers=true])
 
+AS_IF([test x"$enable_naked_pointers_checker" = "xyes" ],
+  [AS_CASE(["$arch","$system"],
+    [amd64,linux|amd64,macosx \
+    |amd64,openbsd|amd64,win64 \
+    |amd64,freebsd],
```

Comment:
I'll double-check, but I think this should work on mingw-w64 amd64 as well (so `amd64,mingw64`)